openhcl/hcl/src/ioctl/snp.rs

```diff
@@ -132,7 +134,8 @@ impl MshvVtl {
     }
 
     /// Gets the current vtl permissions for a page.
-    pub fn rmpquery_page(&self, gpa: u64, vtl: GuestVtl) -> SevRmpAdjust {
+    /// Use with caution: only supported on Genoa+
```

Comment:
So long as this returns a proper error now instead of panicking, is this comment needed?

Reply:
Roman changed the kernel so I think it will return some form of not supported now. But at least for me, it would be helpful to have some words reminding me about where I can use it, especially since you'd actually have to test on Milan to run into a bug.